cdap-ui/pom.xml

```diff
@@ -190,84 +190,68 @@
           <plugin>
             <groupId>com.github.eirslett</groupId>
             <artifactId>frontend-maven-plugin</artifactId>
-            <version>0.0.23</version>
+            <version>1.6</version>
             <executions>
               <execution>
                 <id>dist</id>
                 <goals>
-                  <goal>install-node-and-npm</goal>
+                  <goal>install-node-and-yarn</goal>
                 </goals>
                 <configuration>
-                  <nodeVersion>v4.5.0</nodeVersion>
-                  <npmVersion>3.8.8</npmVersion>
+                  <nodeVersion>v8.7.0</nodeVersion>
+                  <yarnVersion>v1.2.1</yarnVersion>
                 </configuration>
               </execution>
-            </executions>
-          </plugin>
-          <plugin>
-            <groupId>org.codehaus.mojo</groupId>
-            <artifactId>exec-maven-plugin</artifactId>
-            <version>1.3.1</version>
-            <executions>
+
               <execution>
-                <id>npm-install</id>
-                <phase>process-resources</phase>
+                <id>yarn-install</id>
                 <goals>
-                  <goal>exec</goal>
+                  <goal>yarn</goal>
                 </goals>
                 <configuration>
-                  <executable>${project.basedir}/node/node</executable>
-                  <arguments>
-                    <executable>${project.basedir}/node/npm/bin/npm-cli.js</executable>
-                    <argument>install</argument>
-                    <argument>-s --progress=false</argument>
-                  </arguments>
+                  <arguments>install --silent --frozen-lockfile</arguments>
                 </configuration>
               </execution>
+
               <execution>
                 <id>bower-install</id>
-                <phase>process-resources</phase>
                 <goals>
-                  <goal>exec</goal>
+                  <goal>bower</goal>
```

Comment:
This will require bower to be installed globally? For someone who is just building sdk (backend folks) would this fail?

Reply:
Seems to work without bower globally installed